packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -0,0 +1,128 @@
+              >
+                New release
+              </Button>
+              <ModalClose
+                onClick={(event) => {
+                  if (typeof onRequestClose === 'function') {
+                    onRequestClose(event);
+                  }
+
+                  dismissChangelog();
+                }}
+                data-testid="changelogModalClose"
+              />
+            </header>
+            <section className="flex flex-col flex-1 p-5 h-full shrink max-h-full">
+              <Image
+                className="object-cover w-44 h-28 rounded-lg"
+                alt="Post Cover image"
+                src={post.image}
+                fallbackSrc={cloudinary.post.imageCoverPlaceholder}
+                loading="lazy"
+                data-testid="changelogImage"
+              />
+              <h3
+                className="mt-2 font-bold text-theme-label-primary typo-title3"
+                data-testid="changelogTitle"
+              >
+                {post.title}
+              </h3>
+              <time
+                className="text-theme-label-quaternary typo-callout"
+                dateTime={post.createdAt}
+                data-testid="changelogDate"
+              >
+                {postDateFormat(post.createdAt)}
+              </time>
+              <div className="mt-2 w text-theme-label-tertiary typo-callout">
+                {post.summary}
+              </div>
+            </section>
+            <footer className="flex gap-3 items-center py-3 px-4 w-full h-16 border-t border-theme-divider-tertiary">
+              <Button
+                className="btn-tertiary"
+                onClick={dismissChangelog}
+                tag="a"
+                href={post.permalink}
+                data-testid="changelogReleaseNotesBtn"
+              >
+                Release notes
+              </Button>
+              {isExtension && (
+                <Button
+                  className="bg-cabbage-40 btn-primary"
+                  data-testid="changelogExtensionBtn"
+                  onClick={() => {
+                    if (isExtension) {
+                      // @ts-expect-error Property 'browser' does not exist on type 'Window & typeof globalThis'
+                      const sendMessage = window?.browser?.runtime?.sendMessage;
+
+                      if (typeof sendMessage === 'function') {
+                        sendMessage({ type: 'REQUEST_UPDATE' });
+                      }
+                    }
+                  }}
+                >
+                  Update extension
+                </Button>
+              )}
+            </footer>
+          </div>
+        )
+      }
+      offset={[6 * 16, 2.5 * 16]}
+      interactive
+      container={{
+        className: 'shadow',
+        paddingClassName: 'p-0',
+        roundedClassName: 'rounded-16',
+        bgClassName: 'bg-cabbage-40',
+        arrowClassName: styles.changelogTooltipArrow,
```

Comment:
It would actually be much preferred if we won't be using CSS module. What we can probably do is, make the container accept an `arrow: ReactNode` property, so we can just use it if it has value to override the existing arrow.